### ISSUE TITLE
Incompatible with MailFlow plugin

### ISSUE THREAD

**Issue by JayBrown** [robertklep/quotefixformac#62]
There's a great little Mail plugin called MailFlow – https://github.com/arachsys/mailflow –, and QuoteFix doesn't seem to be compatible. MailFlow apparently overrides some (all?) of QuoteFix's functionality. For example: MailFlow (apart from implementing format=flowed) trims the attribution line (removes the annoying date & time), and doesn't insert it in the quote level, but as part of the new message, which is really awesome. QuoteFix now apparently can't set leading whitespace and move cursor to the top of the new message. (Don't know about the other functionality.) But this is just a heads up. When users like myself also use MailFlow, QuoteFix stops working properly.

@arachsys

**Comment by robertklep** [comment]
Yeah, it's going to be tricky to have two plugins active that are (partially) trying to do the same thing.

I don't know exactly how Mail decides the order in which plugins are run, but it's probably fair to assume it's going to be random.

You may be able to get _some_ QF functionality working, but I'm taking your word for it that a lot of functionality will be broken (also because of how Mail plugins work, which is a bit shady due to lack of proper API's).

**Comment by arachsys** [comment]
Robert Klep <notifications@github.com> wrote:

> Yeah, it's going to be tricky to have two plugins active that are  
> (partially) trying to do the same thing.
>
> I don't know exactly how Mail decides the order in which plugins are run,  
> but it's probably fair to assume it's going to be random.
>
> You may be able to get some QF functionality working, but I'm taking your  
> word for it that a lot of functionality will be broken (also because of  
> how Mail plugins work, which is a bit shady due to lack of proper API's).

MailFlow is a single simple .py file, so failing all else you could just  
comment out the bits you don't want (probably the swizzling of the two  
ComposeViewController selectors) to prevent it interfering with QuoteFix?

I suspect QuoteFix probably has a more sophisticated and configurable  
attribution repair system than the super-simple one I wrote for MailFlow  
and MailWrap.

Best wishes,

Chris.
